### ISSUE TITLE
Should the default addon installation blueprint add placeholders for the google api configuration keys?

### ISSUE THREAD

**Issue by begedin** [coderly/ember-gdrive#103]
@venkatd 

While dealing with #100, I got an idea.

What if the default blueprint which runs upon calling `ember install:addon coderly/ember-gdrive` used whichever approach we opted for in #101 to add a placeholder `ember-gdrive` configuration group. It would look something like:

```JavaScript
'ember-gdrive': {
  GOOGLE_API_KEY: ''
  GOOGLE_MIME_TYPE: 'application/',
  GOOGLE_DRIVE_SDK_APP_ID: '',
  GOOGLE_CLIENT_ID: ''
}
```
The user would still have to add their own keys, but everything would be ready for them to do so.

I'm not as much for this idea as I am with #101, but it still might be possible and worth the trouble.

**Comment by venkatd** [comment]
This may be worth it. What's the effort involved to doing this?

You could also write it like this:
```
'ember-gdrive': {
  GOOGLE_API_KEY: '<insert here>'
  GOOGLE_MIME_TYPE: 'application/',
  GOOGLE_DRIVE_SDK_APP_ID: '<insert here>',
  GOOGLE_CLIENT_ID: '<insert here>'
}
```

**Comment by begedin** [comment]
It shouldn't be too much effort. The biggest issue is picking the most reliable way to add it to the config file, since the options are to either append to a file, or insert before or after some string. Something like inserting after 'APP = {',  on a new line might work.

**Comment by venkatd** [comment]
That's fine with me

**Comment by begedin** [comment]
@venkatd: This was implemented in #106, but due to constraints caused by how the `insertIntoFile` method works, it's not as robust as I hoped it would be. I listed the potential issues in the pull request as well as a couple of options we might have.
